### PR TITLE
fix(language-selector): width of web component different than react version

### PR DESCRIPTION
**Pull request by NSDrowned**
### Related Ticket(s)

https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/4888

### Description

The default and short language only selectors where missing styles and the max breakpoint that the react version has. This affected how they looked in tablet and desktop. 

Affected files: packages\web-components\src\components\footer\footer.scss

### Changelog

**New**

- Added the 'max' breakpoint and it's values to both (short and default) selectors

**Changed**

- Updated the 'lg' breakpoints for both (short and default) selectors

### CODE REVIEW

File: packages/web-components/src/components/footer/footer.scss [hunk 1]
```diff
@@ -254,7 +259,14 @@
 
   @include carbon--breakpoint('lg') {
     order: 1;
+    flex: 0 0 25%;
+    max-width: 20rem;
+    min-width: 13.5rem;
```
Comment (kennylam) [inline packages/web-components/src/components/footer/footer.scss:264]: Thanks @NSDrowned! We can avoid hard-coding the width values by using `carbon--mini-units()`.

https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/0038432f7fa8b2596b403a534b2f42c527b3f122/packages/styles/scss/components/footer/_language-selector.scss#L51-L52

Comment (NSDrowned) [reply packages/web-components/src/components/footer/footer.scss:264]: Nice! I didn't know that :)

Comment (jeffchew) [reply packages/web-components/src/components/footer/footer.scss:264]: @NSDrowned I see you resolved this thread but I don't see the change, perhaps forgot to push?

Comment (NSDrowned) [reply packages/web-components/src/components/footer/footer.scss]: @jeffchew forgot to push 🤦

**Comment by jeffchew** [comment]
Thanks @NSDrowned , there looks to be prettier issues now, can you take a look?

**Review by kennylam — APPROVED** [review]
LGTM thanks @NSDrowned!

**Pull request merged**